libs/hmpb/render-backend/src/generate_fixtures.ts

```diff
@@ -89,10 +91,33 @@ function generateSampleElectionFixtures(): void {
   }
 }
 
+function generatePrimaryElectionFixtures(): void {
+  fs.mkdirSync(primaryElectionDir, { recursive: true });
+  const { electionDefinition, mammalParty, fishParty } =
+    primaryElectionFixtures;
+
+  fs.writeFileSync(
+    join(primaryElectionDir, 'election.json'),
+    electionDefinition.electionData
+  );
+
+  for (const partyFixtures of [mammalParty, fishParty]) {
+    const { partyLabel, blankBallot, markedBallot } = partyFixtures;
+    const ballots = {
+      [`${partyLabel}-blank-ballot`]: blankBallot,
+      [`${partyLabel}-marked-ballot`]: markedBallot,
+    } as const;
+    for (const [label, document] of Object.entries(ballots)) {
+      generateBallotFixture(primaryElectionDir, label, document);
+    }
+  }
+}
+
 export function main(): void {
   fs.rmSync(fixturesDir, { recursive: true });
 
   generateAllBubbleBallotFixtures();
   generateFamousNamesFixtures();
   generateSampleElectionFixtures();
+  generatePrimaryElectionFixtures();
```

Comment:
I take it `generateSampleElectionFixtures()` is generating general election fixtures? Might want to rename it `generateGeneralElectionFixtures()` for clarity, since all of these are "sample" elections

Reply:
Yes, I totally agree. If I do that renaming, I'd like to rename the base election fixtures as well.

Though I realize that here I did deviate in the naming of this function from the base election fixture name, which is a bit inconsistent.

Reply:
Will tackle some renaming in a separate PR